app/scripts/provider-transform.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * transform elastic search webpage query to display format.  See data-model.json
+ */
+
+/* globals _, commonTransforms */
+/* exported webpageTransform */
+/* jshint camelcase:false */
+
+/* note lodash should be defined in parent scope, as well as commonTransforms */
+var providerTransform = (function(_, commonTransforms) {
+
+    function getProvider(record) {
+        var person = {};
+        person._id = _.get(record, 'uri');
+        person._type = "provider";
+        person.name = _.get(record, 'name', 'Name N/A');
+        person.ethnicities = _.get(record, 'ethnicity');
+        person.height = _.get(record, 'height');
+        person.weight = _.get(record, 'weight');
+        person.ages = _.get(record, 'age');
+
+        title = (person.name != 'Name N/A')? person.name : "";
+        sep = (title == "")? "": ", ";
```

Comment:
The title and sep variables should be local variables (using "var") instead of global variables.